spec/features/admin/show_spec.rb

```diff
@@ -6,4 +6,12 @@
 
     expect(page).to have_content("Welcome to the Admin Dashboard")
   end
+
+  #user story 20
+  it "has a link to the admin merchant and admin invoice indecies" do
+    visit admin_path
+
+    expect(page).to have_link("Admin Merchants", href: "/admin/merchants")
+    expect(page).to have_link("Admin Invoices", href: "/admin/invoices")
+  end
```

Comment:
Same thing w the routes here. To make the test more robust, I'd also add a click_link and expect the page to be able to redirect to the link in the form of a expect(current_path).to eq( etc)